akka-stream/src/main/scala/akka/stream/impl/streamref/SinkRefImpl.scala

```diff
@@ -181,14 +197,16 @@ private[stream] final class SinkRefStageImpl[In] private[akka] (
 
           completedBeforeRemoteConnected match {
             case OptionVal.Some(scala.util.Failure(ex)) ⇒
-              log.warning("Stream already terminated with exception before remote side materialized, failing now.")
+              log.warning("Stream already terminated with exception before remote side materialized, sending failure: " + ex.getMessage)
```

Comment:
use logging placeholders instead of string concatenation.
`toString` of Exception is often better than `getMessage`, includes exception class

Reply:
Oh, also, don't we have a `warning(ex, message)` nowadays (or is that only in typed)?

Reply:
The overload is only Typed @johanandren :)
Thanks, changed